app/src/main/java/at/aau/edu/wizards/ui/lobby/LobbyViewModel.kt

```diff
@@ -30,17 +30,33 @@ class LobbyViewModel(
         server.stopBroadcasting()
     }
 
+    var numPlayer: Int = 1
+        private set
+
+    private var checkTooManyPlayer = false
+
+    fun getCheckTooManyPlayer(): Boolean {
```

Comment:
remove that function write `private set` in line 37. Exactly same behaviour, that's what I meant.

Reply:
completed